### PR TITLE
add 2022 Computational mathematics slides && notebooks

### PR DESCRIPTION
**Pull request by kalmarek**
@blegat I also have something from the conference (which I believe is better actually), but the slides are not in the framework (make-files etc.) should I add this here as well?

### CODE REVIEW

**Comment by blegat** [comment]
Yes, no need to fit the makefiles workflow

**Comment by kalmarek** [comment]
here they are ;) let me know if you can compile this thing

File: 2022_ICM_Computational_Math/slides/marek.tex [hunk 1]
```diff
@@ -0,0 +1,603 @@
...
+   \begin{bmatrix}
+    x^2 \\
+    xy \\
+    y^2 \\
+   \end{bmatrix}
+   = \mathbf{x}^T P(\lambda) \mathbf{x}
+   \]
+   \end{example}
+   \pause
+   $P$ is so called \textbf{Gramm matrix} for $f$.
```
Comment (blegat) [inline 2022_ICM_Computational_Math/slides/marek.tex]:
```suggestion
   $P$ is so called \textbf{Gram matrix} for $f$.
```

**Comment by blegat** [comment]
Yes, it does compile with the `Makefile` :)

**Comment by kalmarek** [comment]
oh really? for this one https://github.com/blegat/SumOfSquaresSlides/pull/7/files#diff-c05a0fce2fa34adc5a2a2c6662878053e83c381aea74d4bc7cba5217da35cea2
I didn't provide any make files. Does make just run over all `tex` files recursively?

**Comment by blegat** [comment]
Not this one, only the slides

**Pull request merged**